tests/testthat/test-validate_ensemble_inputs.R

```diff
@@ -83,3 +119,8 @@ test_that("weights depend on output_type_id for cdf and pmf output_type generate
       validate_ensemble_inputs(weights = fweight, valid_output_types = c("pmf"))
   )
 })
+
+test_that("validated model_output is a model_out_tbl", {
+  validated_outputs <- validate_ensemble_inputs(model_outputs, valid_output_types = "quantile")[[1]]
+  expect_true(inherits(validated_outputs, "model_out_tbl"))
```

Comment:
https://testthat.r-lib.org/reference/inheritance-expectations.html

```suggestion
  expect_s3_class(validated_outputs, "model_out_tbl")
```

Reply:
This will make it less frustrating in the future if the class name changes and you have to go back and figure out what the test is doing.